[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron. -knative-prow-robot

/cc knative-sandbox/channel-wg-leads
/assign knative-sandbox/channel-wg-leads

Produced via:
```shell
./hack/update-deps.sh --upgrade --release v0.23
./hack/update-codegen.sh
```

Details:
```
bumping knative.dev/eventing 7775e37...5dccb29:
  > 5dccb29 Tidy up the CRD to remove the v1beta1 (# 5298)
  > 978778a Increase Subscription validation test coverage (# 5305)
  > 0d25138 Increase Trigger validation test coverage (# 5306)
  > a62ed6a :broom: lifting some apis to more recent level (# 5300)
  > 6f7c5cf Doing version bumps for some files to v1 (# 5299)
  > 83a1713 Drop --strict flag on ko (# 5303)
  > a0a3302 upgrade to latest dependencies (# 5297)
bumping knative.dev/control-protocol b21afd8...bed7402:
  > bed7402 Conformance test (# 58)
  > 34d4ac5 Send a no payload message (# 57)
  > b5bfcd0 MessageRouter now fails when routing fails (# 56)
  > 3703480 upgrade to latest dependencies (# 59)
bumping knative.dev/hack b6ab329...88c69cd:
  > 88c69cd Add debug info for release.sh to investigate https://knative.slack.com/archives/CA1DTGZ2N/p1619160369065000 (# 66)
  > b5f6e25 Update OWNERS_ALIASES to match autogen in community (# 62)
bumping knative.dev/networking 57bee66...b62df94:
  > b62df94 upgrade to latest dependencies (# 402)
  > 2696fca Add missing JSON tag to protocol type (# 400)
  > 7a7ad51 upgrade to latest dependencies (# 401)
  > 2ee665e upgrade to latest dependencies (# 398)
  > 2b84569 upgrade to latest dependencies (# 397)
bumping knative.dev/pkg 0c5259d...cfc1eed:
  > cfc1eed Better support for error wrapping with ReconcilerEvents (# 2106)
  > 2a0fc65 Make VolatileTime compatible with controller-gen (# 2104)
  > 78b8140 Use semantic equality for status update retries (# 2103)
```

devguyio commented:
/retest

devguyio commented:
/assign

devguyio commented:
/retest

devguyio commented:
/test pull-knative-sandbox-eventing-kafka-integration-test-channel-consolidated-tls

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/eventing 7775e37...5dccb29:
  > 5dccb29 Tidy up the CRD to remove the v1beta1 (# 5298)
  > 978778a Increase Subscription validation test coverage (# 5305)
  > 0d25138 Increase Trigger validation test coverage (# 5306)
  > a62ed6a :broom: lifting some apis to more recent level (# 5300)
  > 6f7c5cf Doing version bumps for some files to v1 (# 5299)
  > 83a1713 Drop --strict flag on ko (# 5303)
  > a0a3302 upgrade to latest dependencies (# 5297)
bumping knative.dev/control-protocol b21afd8...bed7402:
  > bed7402 Conformance test (# 58)
  > 34d4ac5 Send a no payload message (# 57)
  > b5bfcd0 MessageRouter now fails when routing fails (# 56)
  > 3703480 upgrade to latest dependencies (# 59)
bumping knative.dev/hack b6ab329...88c69cd:
  > 88c69cd Add debug info for release.sh to investigate https://knative.slack.com/archives/CA1DTGZ2N/p1619160369065000 (# 66)
  > b5f6e25 Update OWNERS_ALIASES to match autogen in community (# 62)
bumping knative.dev/networking 57bee66...b62df94:
  > b62df94 upgrade to latest dependencies (# 402)
  > 2696fca Add missing JSON tag to protocol type (# 400)
  > 7a7ad51 upgrade to latest dependencies (# 401)
  > 2ee665e upgrade to latest dependencies (# 398)
  > 2b84569 upgrade to latest dependencies (# 397)
bumping knative.dev/pkg 0c5259d...cfc1eed:
  > cfc1eed Better support for error wrapping with ReconcilerEvents (# 2106)
  > 2a0fc65 Make VolatileTime compatible with controller-gen (# 2104)
  > 78b8140 Use semantic equality for status update retries (# 2103)

Signed-off-by: Knative Automation <automation@knative.team>

matzew reviewed (approved):
/lgtm
/approve
